providers/BridgeProvider.tsx

```diff
@@ -27,6 +27,19 @@ const BridgeProvider: FC<{ ethChainId: string }> = ({
 	children,
 	ethChainId,
 }) => {
+	useEffect(() => {
+		async function listenMMAccountChange() {
+			global.ethereum.on("accountsChanged", async function () {
+				const { ethereum } = window as any;
+				const accounts = await ethereum.request({
+					method: "eth_requestAccounts",
+				});
+				setValue({ ...value, Account: accounts[0] });
+			});
+		}
+		listenMMAccountChange();
+	}, []);
+
```

Comment:
needs to tidy up the event listener to avoid memory leak, use `ethereum.removeListener`
and see [here](https://reactjs.org/docs/hooks-effect.html#example-using-hooks-1) for `useEffect` cleanup function

Reply:
sweet should be fixed now